src/app/items/containers/item-permissions/item-permissions.component.ts

```diff
@@ -90,4 +100,33 @@ export class ItemPermissionsComponent implements OnChanges {
       this.changed.emit();
     }
   }
+
+  grantAccess(): void {
```

Comment:
"grantAccess" may be confusing with the other permission button. Call it "grantViewContentAccess" to be more explicit.

Reply:
Fixed